backend/app/api/endpoints/kind/common.py

```
@@ -0,0 +1,286 @@
+            resource['metadata'] = {}
+        
+        # Set namespace
+        resource['metadata']['namespace'] = namespace
+        
+        # Set name if provided (for update operations)
+        if name:
+            resource['metadata']['name'] = name
+        
+        # Validate resource
+        validated_resource = schema_class.parse_obj(resource).dict()
+        return validated_resource
```

Ruff (0.14.2)

Consider moving this statement to an `else` block

(TRY300)